Logo/Logo for Painless Music Player

```diff
@@ -0,0 +1 @@
+
```

Comment:
I don't see any purpose of "Create Folder" commit and this empty file.
Also, this file is not removed.

Reply:
Sorry, I forgot about it